doc/neps/nep-0023-backwards-compatibility.rst

```diff
@@ -257,34 +303,27 @@ ecosystem - being fairly conservative is required in order to not increase the
 extra maintenance for downstream libraries and end users to an unacceptable
 level.
 
-**Semantic versioning.**
-
-This would change the versioning scheme for code removals; those could then
-only be done when the major version number is increased.  Rationale for
-rejection: semantic versioning is relatively common in software engineering,
-however it is not at all common in the Python world.  Also, it would mean that
-NumPy's version number simply starts to increase faster, which would be more
-confusing than helpful. gh-10156 contains more discussion on this alternative.
-
 
 Discussion
 ----------
 
-TODO
-
-This section may just be a bullet list including links to any discussions
-regarding the NEP:
-
-- This includes links to mailing list threads or relevant GitHub issues.
+- `Mailing list discussion on the first version of this NEP in 2018 <https://mail.python.org/pipermail/numpy-discussion/2018-July/078432.html>`__
 
 
 References and Footnotes
 ------------------------
 
-.. [1] TODO
+- `Issue requesting semantic versioning <https://github.com/numpy/numpy/issues/10156>`__
+
+- `PEP 387 - Backwards Compatibility Policy <https://www.python.org/dev/peps/pep-0387/>`__
+
+.. [1] https://searchcode.com/
```

Comment:
Cool, just curious, have you used this: https://sourcegraph.com/search before? This seems nicer, but not sure how they compare.

Reply:
I have, but I use both rarely so don't have a strong preference between the two. 

Will add the reference.